### DISCUSSION TITLE
I need help.

### DISCUSSION THREAD

**Discussion by SackyyPL** [ORelio/Minecraft-Console-Client#2092] [Q&A]
Let's start with the fact that I used MCC a long time ago, and now everything has changed.
I fired up the file "MinecraftClient.exe" and a file named "MinecraftClient.ini" was created for me.
Everything I set up looks like this:
login=xSacky
password=mypassword
serverip=craftplay.pl
type=mojang
method=mcc
I didn't change anything else.
When I then click on the exe file it won't connect me, what am I doing wrong or what else should I do?
Please help =)

**Comment by BruceChenQAQ** [comment]
Try filling in the login field with the email address of the account.

**Reply by SackyyPL** [reply]
> Try filling in the login field with the email address of the account.

still the same :/

**Reply by BruceChenQAQ** [reply]
Can you provide a screenshot when running "MinecraftClient.exe"?

**Reply by SackyyPL** [reply]
sure, I tried connecting to other servers, the same on all of them.
![unknown](https://user-images.githubusercontent.com/110846389/183741324-24893ca4-f685-45ce-87b7-f05aef0e72ec.png)

**Reply by ReinforceZwei** [reply]
Mojang account cannot play online server anymore. You have to migrate your account.

**Reply by BruceChenQAQ** [reply]
If the account type configured in the settings is "mojang", there should be a log like this: "Connecting to Minecraft.net..." (not Microsoft)

I don't have an unmigrated mojang account so I can't verify this, but in version 1.19 and above, mojang accounts are not able to get the keys needed for message signing.

**Reply by SackyyPL** [reply]
I changed the "type=" option to "microsoft" and suddenly it worked, thank you for your help 👍

**Reply by SackyyPL** [reply]
because I had my account migrated to microsoft.

**Reply by SackyyPL** [reply]
I currently have another problem, after joining the server after a few moments I get disconnected. It is like this all the time.
![image](https://user-images.githubusercontent.com/110846389/183830890-6a9dee9e-0a53-47c7-a08a-916aa63a16d4.png)

**Reply by milutinke** [reply]
> I currently have another problem, after joining the server after a few moments I get disconnected. It is like this all the time. ![image](https://user-images.githubusercontent.com/110846389/183830890-6a9dee9e-0a53-47c7-a08a-916aa63a16d4.png)

Try my fork of the MCC, it has that problem fixed
https://github.com/milutinke/Minecraft-Console-Client
